tests/e2e/visual/targets/targets-overview.wdio-spec.js

```diff
@@ -0,0 +1,694 @@
+    await commonPage.openFastActionReport('pregnancy');
+    await pregnancyForm.submitDefaultPregnancy();
+    await commonPage.openFastActionReport('delivery');
+    //delivery form creation
+    await commonEnketoPage.selectRadioButton('What is the outcome for the woman?', 'Alive and well');
+    await genericForm.nextPage();
+    await dangerSignPage.selectAllDangerSignsDelivery('No');
+    await genericForm.nextPage();
+    await commonEnketoPage.selectRadioButton('How many babies were delivered?', '1');
+    await commonEnketoPage.selectRadioButton('How many babies are alive?', '1');
+    await commonEnketoPage.selectRadioButton('Where did delivery take place?', 'Health facility');
+    await commonEnketoPage.selectRadioButton('How did she deliver?', 'Vaginal');
+    await commonEnketoPage.setDateValue('Date of delivery', BABY_DOB);
+    await genericForm.nextPage();
+    await commonEnketoPage.selectRadioButton('What is the condition of baby?', 'Alive and well');
+    await commonEnketoPage.setInputValue('Name', BABY_NAME);
+    await commonEnketoPage.selectRadioButton('Sex', 'Male');
+    await commonEnketoPage.selectRadioButton('Birth weight', 'I don\'t know');
+    await commonEnketoPage.selectRadioButton('Birth length', 'I don\'t know');
+    await commonEnketoPage.selectRadioButton('What vaccines have they received?', 'None');
+    await commonEnketoPage.selectRadioButton('Is the child exclusively breastfeeding?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Were they initiated on breastfeeding within on hour of delivery?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Infected umbilical cord', 'No');
+    await commonEnketoPage.selectRadioButton('Convulsions', 'No');
+    await commonEnketoPage.selectRadioButton('Difficulty feeding or drinking', 'No');
+    await commonEnketoPage.selectRadioButton('Vomits everything', 'No');
+    await commonEnketoPage.selectRadioButton('Drowsy or unconscious', 'No');
+    await commonEnketoPage.selectRadioButton('Body stiffness', 'No');
+    await commonEnketoPage.selectRadioButton('Yellow skin color', 'No');
+    await commonEnketoPage.selectRadioButton('Fever', 'No');
+    await commonEnketoPage.selectRadioButton('Blue skin color (hypothermia)', 'No');
+    await genericForm.nextPage();
+    await genericForm.nextPage();
+    await commonEnketoPage.selectCheckBox('Which PNC visits have taken place so far?', 'None of the above');
+    await genericForm.nextPage();
+
+    const summaryTexts = [
+      pregnantWoman2.name,
+      '25', //age
+      'Alive and well', //woman's condition
+      BABY_DOB,
+      'Health facility' //delivery place
+    ];
+
+    await commonEnketoPage.validateSummaryReport(summaryTexts);
+    await genericForm.submitForm();
+  
+    await commonPage.goToPeople(pregnantWoman._id);
+    await commonPage.openFastActionReport('pregnancy_home_visit');
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+
+
+  }); 
```

Comment:
For delivery reports, instead of using the UI to create the delivery, you can use the [deliveryFactory](https://github.com/medic/cht-core/blob/908fa932f3c9e17a1af270675ab73d79326f415e/tests/factories/cht/reports/delivery.js#L1), as shown in [this example (line 19)](https://github.com/medic/cht-core/blob/908fa932f3c9e17a1af270675ab73d79326f415e/tests/e2e/default-mobile/navigation/navigation.wdio-spec.js#L19).
You can generate the data in the describe block and then save it in the before block as well as the persons and other reports.